requirements-python3.txt

```diff
@@ -1,3 +1,4 @@
 defusedxml>=0.5.0rc1
 python3-openid>=3.0.10
+pycryptodome>=3.4.7
```

Comment:
Where's this used?

Reply:
It's being used for RSA1 encryption.

Reply:
It is an external dependency which oauthlib isn't bundled with.